app/react/V2/Routes/Settings/ParagraphExtraction/components/ExtractorModal.tsx

```diff
@@ -118,13 +122,17 @@ const ExtractorModal = ({
 
         <div className="flex flex-col">
           <div className="flex justify-center w-full gap-2">
-            {/* duplicate structure, can be a function */}
             <div className={`w-2 h-2 rounded-full ${isActiveStepClassName(step === 1)}`} />
             <div className={`w-2 h-2 rounded-full ${isActiveStepClassName(step === 2)}`} />
           </div>
           {templateToOptions.length !== 0 && step === 1 && (
             <span className="mt-5 text-gray-500 font-light text-sm">
-              <Translate>Templates meeting required criteria</Translate>
+              <Translate>Templates meeting</Translate>{' '}
+              <Link to={linkPXTemplateCriteria} target="_blank">
+                <span className="underline">
+                  <Translate>required criteria</Translate>
```

Comment:
You can add classname to Translate like : `<Translate className="underline">required criteria</Translate>`
Also dividing a string in two might be problematic  when translating